### ISSUE TITLE
FR: Add QA checks to portage (?)

### ISSUE THREAD

**Issue by jonasstein** [mm1ke/gentoo-scripts#29]
Perhaps it is a good idea to add some of the checks to portage:
https://github.com/gentoo/portage/tree/master/bin/postinst-qa-check.d

I put this in the ticket for later, so that the idea does not get lost.

**Comment by simonvanderveldt** [comment]
Wouldn't it make sense to add the majority of these checks to repoman instead of portage?

Asking because I'd like to run these scripts against my overlay, which I can do when changing some some variables (`PORTTREE` in most scripts and `projxml` in `funcs.sh`). One solution to this would be to make these scripts work for overlays. Another would be to add the checks to repoman.
What do you think?

**Comment by jonasstein** [comment]
absolutely, I used portage as synonym for repoman/portage. It was one project once. I think some tests are not implemented in repoman, but in portage, like the current fdo test.

**Comment by simonvanderveldt** [comment]
OK, thx for the clarification!

Whilst I have no experience with the repoman codebase if you guys want I would like to help improve repoman and incorporate these changes.
Is there already a "plan of attack" on how to do so?

**Comment by jonasstein** [comment]
@simonvanderveldt can we meet on IRC in 1h?

**Comment by simonvanderveldt** [comment]
@jonasstein sure! I'm in `#gentoo-dev-help`, if you want another channel or PM just let me know. Same nick as here
